core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -4261,6 +4271,11 @@ private void removeBlocks(List<Long> blocks) throws IOException {
     throw new IOException("Failed to remove deleted blocks from block master", lastThrown);
   }
 
+  @Override
+  public CallTracker composeCallTracker(CallTracker transportTracker) {
```

Comment:
This is allowed to be `@Nullable`, right? We should handle that case.

Reply:
removed nullable.